scripts/teardown.sh

```diff
@@ -31,4 +31,8 @@ source "$ROOT/scripts/common.sh"
 
 # Tear down Terraform-managed resources and remove generated tfvars
 cd "$ROOT/terraform" || exit; terraform destroy -input=false -auto-approve
+rm -rf "$ROOT/terraform/.terraform"
```

Comment:
`terraform destroy` does not remove these?

Reply:
I've had to manually remove this each time I want to re-run `make create` as a new deployment, but I can test again to verify.

Reply:
Here's the terraform directory listing before (technically right after starting) and after running `make teardown`:
```console
→ ll terraform/
total 128
drwxr-xr-x  11 mmatthias  primarygroup   352B Nov 12 10:39 .
drwxr-xr-x  13 mmatthias  primarygroup   416B Nov 12 10:17 ..
drwxr-xr-x   3 mmatthias  primarygroup    96B Nov 12 10:19 .terraform
-rw-------   1 mmatthias  primarygroup   235B Nov 12 10:39 .terraform.tfstate.lock.info
-rw-r--r--   1 mmatthias  primarygroup   718B Nov 12 10:06 outputs.tf
-rw-r--r--   1 mmatthias  primarygroup   632B Nov 12 10:06 provider.tf
-rw-r--r--   1 mmatthias  primarygroup   2.1K Nov 12 10:18 resources.tf
-rw-r--r--   1 mmatthias  primarygroup    16K Nov 12 10:39 terraform.tfstate
-rw-r--r--   1 mmatthias  primarygroup    17K Nov 12 10:39 terraform.tfstate.backup
-rw-r--r--   1 mmatthias  primarygroup    57B Nov 12 10:19 terraform.tfvars
-rw-r--r--   1 mmatthias  primarygroup   891B Nov 12 10:06 variables.tf

 → ll terraform/
total 80
drwxr-xr-x   9 mmatthias  primarygroup   288B Nov 12 10:44 .
drwxr-xr-x  13 mmatthias  primarygroup   416B Nov 12 10:17 ..
drwxr-xr-x   3 mmatthias  primarygroup    96B Nov 12 10:19 .terraform
-rw-r--r--   1 mmatthias  primarygroup   718B Nov 12 10:06 outputs.tf
-rw-r--r--   1 mmatthias  primarygroup   632B Nov 12 10:06 provider.tf
-rw-r--r--   1 mmatthias  primarygroup   2.1K Nov 12 10:18 resources.tf
-rw-r--r--   1 mmatthias  primarygroup   318B Nov 12 10:44 terraform.tfstate
-rw-r--r--   1 mmatthias  primarygroup    17K Nov 12 10:39 terraform.tfstate.backup
-rw-r--r--   1 mmatthias  primarygroup   891B Nov 12 10:06 variables.tf
```

Reply:
(ll = 'ls -ahl')